lib/libprocstat/libprocstat.c

```diff
@@ -2516,7 +2516,7 @@ is_elf32_sysctl(pid_t pid)
 {
 	int error, name[4];
 	size_t len, i;
-	static char sv_name[256];
+	static char sv_name[32];
```

Comment:
I also don't see why this is `static`, this is library code and it's not thread-safe.

Reply:
Indeed, that's nonsense.